### PR TITLE
Backbone 373 display result

### PR DESCRIPTION
**Pull request by P-DAO**

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -23,3 +23,53 @@ code {
...
+}
+[role="region"][aria-labelledby][tabindex] {
+    width: 100%;
+    max-height: 98vh;
+    overflow: auto;
+}
+[role="region"][aria-labelledby][tabindex]:focus {
+    box-shadow: 0 0 0.5em rgba(0, 0, 0, 0.5);
+    outline: 0;
+}
```
Comment (julienleows) [inline frontend/src/index.css:75]: Can you add spaces between each css class.
Can you move your css part into your component folder.

File: frontend/src/components/ClericalReview/TableResult.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    mpr_fprKeys.map(mpr => (
+                        <tr key={mpr}>
+                            <th key={mpr}>{mpr}</th>
+                        </tr>
+                    ))
+                }
+                </tbody>
+            </table>
+        </div>
+
```
Comment (julienleows) [inline frontend/src/components/ClericalReview/TableResult.js:64]: You can remove this space.

File: frontend/src/components/ClericalReview/TableResult.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+/**
+ * Represents a table component displaying data from mpr_fpr and reg_pmp arrays.
+ *
```
Comment (julienleows) [inline frontend/src/components/ClericalReview/TableResult.js:3]: You can remove this space.

File: frontend/src/App.js [hunk 4]
```diff
@@ -10,7 +11,11 @@ function App() {
             <Route path="/" element={<MainPage />} />
 
             {/* Define a route for display results */}
-            <Route path="/configResearch" element={<FormSearchConfig/>} />
+            <Route path="/configResearch" element={<FormSearchConfig />} />
+
+            {/* Define a route for result table */}
+            <Route path="/result" element={<TableResult/>} />
+
```
Comment (julienleows) [inline frontend/src/App.js:18]: You can remove this space.

**Review by julienleows — CHANGES_REQUESTED** [review]

**Review by P-DAO — COMMENTED** [review]
c'est fait je te renvoie ça

**Pull request closed without merging**